providers/summary/scancode.js

```diff
@@ -85,7 +86,7 @@ class ScanCodeSummarizer {
     const list = setToArray(licenses)
     if (!list) return null
     return list
-      .map(normalizeSpdx)
+      .map(SPDX.normalize)
       .filter(x => x)
       .join(' AND ')
```

Comment:
the work to make this right is in #322 

Once we get through this piece here, I can update that branch